src/System.Globalization/ref/System.Globalization.Forwards.cs

```diff
@@ -5,3 +5,9 @@
 // These two types were moved down to System.Runtime
 [assembly: System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Globalization.Calendar))]
 [assembly: System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Globalization.CalendarWeekRule))]
+[assembly: System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Globalization.CompareOptions))]
```

Comment:
In order to be able to resolve all of these new forwards, you will need to add a reference to System.Runtime.pkgproj to the ref/System.Globalization.csproj. Alternatively you can keep these guys in System.Globalization temporarily until we publish a System.Runtime that has your change, and then remove them here.